gadget/gadget.go

```diff
@@ -426,6 +438,9 @@ func validateVolume(name string, vol *Volume, model Model) error {
 
 	state := &validationState{}
 	previousEnd := quantity.Size(0)
+	// TODO: should we also validate that if there is a system-recovery-select
+	// role there should also be at least 2 system-recovery-image roles and
+	// same for system-boot-select and at least 2 system-boot-image roles?
```

Comment:
that sounds like a good idea

Reply:
To be clear do you want this as part of this PR or just to do eventually?

Reply:
can be done separately but sounds a good thing to have to dot the feature i's

Reply:
ok, I will see what I can do, if it's a lot of work I may defer it to not this PR

Reply:
I think for a followup PR